dex/networks/eth/doc.go

```diff
@@ -0,0 +1,13 @@
+//go:build gpl
+// +build gpl
+//
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+//
+// This package also imports go-ethereum code and so carries the burden of
+// go-ethereum's GNU Lesser General Public License.
+
+// Package eth implements methods to work with ethereum swap contracts and
+// transactions. The code is unique as it requires a more restrictive liscense
+// and connot be used in proprietary software.
```

Comment:
I'm not totally sure of the wording here.  I know GPL code is used all the time in closed source systems, and it's not quite as simple as this.  I'm pretty sure some the requirements are distributing the license, possibly the gpl code or a link to the gpl code, maybe the modifications, it all depends.  Whatever the specifics, there are a lot more strings attached.  https://opensource.stackexchange.com/questions/2338/can-i-use-gpl-libraries-in-a-closed-source-project-if-only-the-output-is-distrib  

The developers of the proprietary software can sort out the details for their product. So, I think we should change the wording from:

> The code is unique as it requires a more restrictive liscense and connot be used in proprietary software

to

> The LGPL is a more restrictive license that may be more of a burden for closed source software. As such, the "lgpl" build tag is required for this package.

Reply:
Changed to the suggested wording.